src/api/emit.js

```diff
@@ -25,5 +30,8 @@ export default function (elem, name, opts = {}) {
   if (opts.cancelable === undefined) {
     opts.cancelable = true;
   }
+  if (opts.composed === undefined) {
```

Comment:
@bradleyayers Not sure it's officially documented yet. The only real evidence of agreement we have is https://github.com/w3c/webcomponents/issues/513 and the fact that it landed in v1 stable in both Blink and Webkit. They mention it in the [DOM standard](https://dom.spec.whatwg.org) but it's somewhat out of context. I also [asked Rob Dodson](https://twitter.com/treshugart/status/772656495167086593) about it and he gave a link to https://developers.google.com/web/fundamentals/primers/shadowdom/#customevents which is close to what we're doing here.

Though it would seem that it's nice to not have it bubble out by default, I would think this would be considered breaking for Skate, so maybe we can invert this in 2.0? If so, I'll create an issue for that.